bot/bot_test.ttl

```diff
@@ -0,0 +1,114 @@
+@prefix : <http://bot_test#> .
+@prefix cc: <http://creativecommons.org/ns#> .
+@prefix bot: <https://w3id.org/bot#> .
+@prefix dbo: <http://dbpedia.org/ontology/> .
+@prefix dce: <http://purl.org/dc/elements/1.1/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix bot_test: <http://bot_test#> .
+@base <http://bot_test> .
```

Comment:
reminder for a next commit:

1. bot_test is not valid URL host name, should be ex. http://example.org/bot#
2. use a URI that ends with # or /, maybe the same as bot_test ?